corehq/apps/users/models.py

```diff
@@ -230,6 +230,11 @@ class UserRole(Document):
     permissions = SchemaProperty(Permissions)
     is_archived = BooleanProperty(default=False)
 
+    @classmethod
+    @quickcache(['doc_id'], timeout=0)
+    def get(cls, doc_id):
```

Comment:
(Nothing actionable there, was just bringing that up since I'd just spent some time looking at it.)